plato/draw/zdog/Box.py

```diff
@@ -1,5 +1,18 @@
+import numpy as np
 from ... import draw
 from .Lines import Lines
 
-class Box(draw.Box, Lines):
+class Box(draw.Box):
     __doc__ = draw.Box.__doc__
```

Comment:
Do the other backends need to set `__doc__ = draw.Box.__doc__`? It's not consistent, though I haven't checked the way sphinx renders it yet.